2/Dockerfile

```diff
@@ -65,6 +65,11 @@ ENV NODE_VERSION 8.7.0
 RUN wget http://nodejs.org/dist/v0.10.30/node-v0.10.30-linux-x64.tar.gz
 RUN tar --strip-components 1 -xzvf node-v* -C /usr/local
 
+# Setup zabbix_sender for monitoring
+# Installation commands from  https://www.zabbix.com/download?zabbix=3.4&os_distribution=centos&os_version=7&db=MySQL
+RUN rpm -i http://repo.zabbix.com/zabbix/3.4/rhel/7/x86_64/zabbix-release-3.4-2.el7.noarch.rpm && \
+    yum install -y zabbix-agent
+
```

Comment:
Does running `yum clean all` in the same RUN statement bring any noticeable change to the image?
How does the agent get started?

On a different note, the node-v0.10 that gets downloaded needs to be deleted as well unless we are making use of multistage build

Reply:
@sthaha A yum clean should help, updating the PR.  We could clean this up in another PR to install all system packages in one step making the overall build faster followed by one single clean.

I'm not sure if node is used. Ideally not in this PR. 

The agent; you caught me there. Somehow I send the wrong patch. The package to install is zabbix_sender and its a CLI tool that will be called from Jenkins. No need to run an agent. Updating PR.